migrator/migrations/n_04_to_n_05_postgres_images/postgres/store.go

```diff
@@ -454,11 +418,6 @@ func copyFromImageCVEEdges(ctx context.Context, tx *postgres.Tx, iTime *protoTyp
 		ids.Add(obj.GetId())
 	}
 
-	// Remove orphaned edges.
-	if err := removeOrphanedImageCVEEdges(ctx, tx, objs[0].GetImageId(), ids.AsSlice()); err != nil {
-		return err
-	}
-
 	exisitingEdgeIDs, err := getImageCVEEdgeIDs(ctx, tx, objs[0].GetImageId())
```

Comment:
This is not needed in migration because we do not need to reconcile. What we read from DB goes back as is.